dev-python/python-evdev/Manifest

```diff
@@ -1 +1 @@
-DIST python-evdev-1.1.2.tar.gz 83067 BLAKE2B 6434d5c62e4a072928d555bfc92e94e7f63ef98ebbf9cd95795a92d6ccdf7b5d4b0f3133a4b3f2524083b842f78ccc305a5da13af59052be3724d83ea4039328 SHA512 054b2e5af532079dc52919d90791d12d7caf75cade07d18b94ea519869659413ea059570b16ebb236a715c541b17016db380bea4e3c1cd21f96824c6303b45d6
+DIST python-evdev-1.3.0.tar.gz 85018 BLAKE2B ba5c391631ecf8cc0a46f21830209c59b1217940ff25e051e027bc8896d71965a394f222f4e3d629ba2a2f8056ccbdc0dfaf43a37684d7157c7456d61f2d8314 SHA512 392fb4fabed344e6db4e257d4cfc7043393efda15d940be80b013e4c4e350334148a7b04aa0a7239f824ad2c7e0bb057c5cff1c0f320d30b25b0c9aab65a6891
```

Comment:
You can't do a version bump like this. You need to make one commit for adding a new file, and 2nd commit for removing one.